include/bluetooth/mesh/light_ctrl_srv.rst

```diff
@@ -0,0 +1,427 @@
+
+* The :ref:`bt_mesh_onoff_cli_readme` model -- which should publish to the Light LC Server's extended Generic OnOff Server model.
+* The :ref:`bt_mesh_light_ctrl_cli_readme` model -- which should publish Light OnOff Set messages to the Light LC Server.
+
+Occupancy sensors may also trigger the On event, depending on the current state and occupancy mode:
+
+* If the occupancy mode is enabled, sensor readings that indicating activity can trigger an On event at any time.
+* If the occupancy mode is disabled, sensors cannot turn the lights on, but they will still prevent lights from turning off.
+
+The following sensor types may also trigger the On event:
+
+Motion sensed : :cpp:var:`bt_mesh_sensor_motion_sensed`
+   Any sensor value higher than 0 triggers an On event in the Light LC Server state machine.
+   Messages with a value of 0 are ignored.
+
+People count : :cpp:var:`bt_mesh_sensor_people_count`
+   Any sensor value higher than 0 triggers an On event in the Light LC Server state machine.
+   Messages with a value of 0 are ignored.
+
+Presence detected : :cpp:var:`bt_mesh_sensor_presence_detected`
+   Messages with a ``true`` value triggers an On event in the Light LC Server state machine.
+   Messages with a ``false`` value are ignored.
+
+Time since motion sensed : :cpp:var:`bt_mesh_sensor_time_since_motion_sensed`
+   When the sensor's *Time since motion sensed* value is lower than the Light LC Server's occupancy delay, the Light LC Server starts a timer that expires at the time equal to motion sensed time plus occupancy delay.
+   When this timer expires, an On event is generated.
+
+.. note::
+   Only sensors reporting the *Time since motion sensed* type will be affected by the occupancy delay.
+   Other sensor triggers are always instantaneous.
+
+Off event
+~~~~~~~~~
+
+The Off event can only be generated by a light switch being turned off.
+It moves the Light LC Server into Standby, transitioning from the previous light level with the
+:cpp:enumerator:`manual mode Standby fade time <bt_mesh_light_ctrl::BT_MESH_LIGHT_CTRL_PROP_TIME_FADE_STANDBY_MANUAL>` .
+
+The Off event puts the Light LC Server into manual mode, which disables sensor input until the :option:`manual mode timeout <CONFIG_BT_MESH_LIGHT_CTRL_SRV_TIME_MANUAL>` expires.
+This prevents the lights from turning back on from the movement of the person pressing the light switch.
+
+.. note::
+   Contrary to the other timing parameters, the manual mode timeout is proprietary, and cannot be reconfigured at runtime by other models in the mesh network.
+
+Light switches may either implement a :ref:`bt_mesh_onoff_cli_readme` or a :ref:`bt_mesh_light_ctrl_cli_readme` to send Off messages turning the Light LC Server on. The Generic OnOff Client model should publish to the Light LC Server's extended Generic OnOff Server model. The Light Lightness Control Client model should publish Light OnOff Set messages to the Light LC Server.
```

Comment:
Please double-check the rst files for lines with multiple sentences.
NCS documentation guidelines (see styleguide) requires 1 sentence per line. I've tried to edit this when reviewing, but apparently missed some lines.